FizzBuzz/src/main/java/rocks/kata/FizzBuzzStringBuilder.java

```diff
@@ -0,0 +1,20 @@
+package rocks.kata;
+
+public class FizzBuzzStringBuilder implements FizzBuzz {
```

Comment:
Myląca nazwa - brzmi bardziej jak klasa, która może być używana we wszystkich implementacjach do tworzenia stringa FizzBuzz, a faktycznie to implementacja FizzBuzza opierająca się o algorytm z wartościami modulo.